examples/toy-model-stochastic-degradation.ipynb

```diff
@@ -0,0 +1,186 @@
+{
```

Comment:
Yep, agreed with Michael here. Looking great! A few things (all minor):

- Can you change the code so that the mean  n_0exp[-kt] renders ok in the notebook? For some reason, this is not displaying as math.
- Once the variance is estimated and included as a function, can we overlay this (or, probably better, the standard deviation) on the plot with the mean to show that most trajectories are within this envelope? Alternatively, if this is too messy, you could plot it separately.